requirements/test.txt

```diff
@@ -0,0 +1,5 @@
+pytest
+pytest-timeout
+pytest-cov
+neon-lang-plugin-libretranslate
```

Comment:
is this needed? it is no longer a default anywhere is it?

Reply:
If that’s what superseded it, not needed. If that’s another name for the import, still needed. Hard to tell from a search.

Reply:
thats just the entrypoint definition, not related to this

Reply:
> is this needed? it is no longer a default anywhere is it?

Just copied it from the GHA definition; lets see if tests still pass without it